diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+users/Graces-MacBook-Pro~/wdi/try
```

Comment:
Whoops, missed this. That `~` isn't part of the path, it's a shortcut for 'home' (i.e. `/Users/Graces-MacBook-Pro`)